### DISCUSSION TITLE
Access right issue

### DISCUSSION THREAD

**Discussion by asitemade4u** [prodrigestivill/docker-postgres-backup-local#99] [Q&A] [answered]
I have been using your container for several months but recently I began having this kind of errors:

```
- 2021/04/28 08:37:03 3539 cmd: /backup.sh
- 2021/04/28 08:37:03 3539: Creating dump of strapi database from 192.168.1.40...
- 2021/04/28 08:37:03 3539: pg_dump: error: query failed: ERROR:  permission denied for table core_store
pg_dump: error: query was: LOCK TABLE public.core_store IN ACCESS SHARE MODE
- 2021/04/28 08:37:03 3539 Exit Status: 1
```

I checked the credentials to the database and the rest of the `docker-compose.yml` file and everything seems fine.
I guess this errors is not strictly linked to the container but can you direct me to a solution?
Best,
Stephen

**Comment by prodrigestivill** [comment, marked as answer]
This is not related with this program.

Please search you error using your favorite web search engine.

A quick search gave me [this thread](https://www.postgresql.org/message-id/20060506015742.GG12810@surnet.cl), please only report issues related with this software.
